myhpom/models/cloudfactory.py

```diff
@@ -0,0 +1,117 @@
+
+import json
+import datetime
```

Comment:
unused import.

Reply:
oh, flake8, right

Reply:
BTW, we didn't use flake8 early in the project, so there are a lot of other warnings. I think I'll go ahead and clean them all up in the myhpom app.